README.md

```diff
@@ -72,6 +72,10 @@ prerequisites for building yosys:
 Similarily, on Mac OS X Homebrew can be used to install dependencies:
 
 	$ brew tap Homebrew/bundle && brew bundle
+	
+The brew bundle is not currently working on Catalina, so do:
+
+	$ brew install python3 bison flex readline gawk libffi git graphviz pkgconfig boost zlib tcl tcl-tk
```

Comment:
Can you change pkgconfig -> pkg-config (since first is just alias)
xdot is missing in list (required for show command)
Also please do update Brewfile to reflect addition of boost (needed for pyosys)

Guess tcl and zlib are actually not needed to be there, since they will anyway be installed by other requirements, can you confirm that.

Reply:
Oh this one is on me.
``$brew bundle`` didn't work because I wasn't inside ``/yosys``
SMH - staying up late is not good.

Perhaps it might be a good idea to add ``$cd yosys`` before  ``brew bundle`` to the README for programmers like me who stay up past their bedtime?

Things work fine on Catalina :)
The brewfile is fine.